tests/Fixtures/Integration/priority/single_expression_per_line,statement_indentation.test

```diff
@@ -0,0 +1,19 @@
+--TEST--
+Integration of fixers: single_expression_per_line,statement_indentation.
+--RULESET--
+{ "single_expression_per_line": true, "statement_indentation": true}
+--EXPECT--
+<?php
+if (
+    $a
+    || $b
+) {
+    // foo
+}
+
+--INPUT--
+<?php
+if ($a
+    || $b) {
+    // foo
+}
```

Comment:
I think the test will pass if you write:
```suggestion
if ($a
|| $b) {
    // foo
}
```
This is because currently, `statement_indentation` handles `$a || $b` as a single statement, so line `|| $b` is not fixed but simply adjusted with the same indentation delta as line `$a`, i.e. 4 spaces.

Reply:
Yes, but since most of the existing code base are using
```
if ($a
    || $b) {
    // foo
}
```
I think, we must fix this correctly.

That's why I consider https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/issues/7884 as a blocker.
Imho this PR is ready only after this test (untouched) is finally green.